libs/ui/src/components/atoms/helperTexts/helperTexts.tsx

```diff
@@ -0,0 +1,59 @@
+import { ParXs } from '../typography';
+import { IconType } from 'react-icons';
+import { WithIcon } from './helperTextStyles';
+import React from 'react';
+import { useTheme } from 'styled-components';
+import { Theme } from '../../../types/theming';
+
+import { BiCheckCircle, BiErrorCircle } from 'react-icons/bi';
+
+export type HelperTextType = {
+  color: string;
+  icon?: IconType;
+  children: React.ReactNode;
+};
+export type SpecialHelperText = {
+  children: React.ReactNode;
+};
+
+export const HelperText = ({ color, icon, children }: HelperTextType) => {
```

Comment:
I guess ultimately size will be a prop for these

Reply:
I don't think so...I'm assuming all input helper texts will be the same. If not we can pass in a text that we'd like to use. Similar to what we do with the icon.

Reply:
Okay, good to know thanks